src/Components/MedicalCase/System.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * The external imports
+ */
+import React from 'react'
+import { View } from 'react-native'
+import { useSelector } from 'react-redux'
+
+/**
+ * The internal imports
+ */
+import { useTheme } from '@/Theme'
+import { SectionHeader, Question } from '@/Components'
+import { translate } from '@/Translations/algorithm'
+
+const System = ({ systemName }) => {
+  const { Gutters } = useTheme()
+  const systemsTranslations = useSelector(
+    state => state.algorithm.item.config.systems_translations,
+  )
+
+  const systemData = useSelector(
+    state => state.questionsPerSystem.item.medicalHistory[systemName],
+  )
+  if (systemData?.length > 0) {
+    return (
```

Comment:
Tu veux pas plutot faire un ternaire like we've done everywhere else ?

`return systemData?.length > 0 ? (.....) : null`